rest/config.go

```diff
@@ -1046,6 +1047,58 @@ func RunServer(config *ServerConfig) {
 	config.Serve(*config.Interface, CreatePublicHandler(sc))
 }
 
+func validateServerContext(sc *ServerContext) (errors []error) {
+	bucketUUIDToDBContext := make(map[string][]*db.DatabaseContext, len(sc.databases_))
+	for _, dbContext := range sc.databases_ {
+		if uuid, err := dbContext.Bucket.UUID(); err == nil {
+			bucketUUIDToDBContext[uuid] = append(bucketUUIDToDBContext[uuid], dbContext)
+		}
+	}
+	sharedBuckets := sharedBuckets(bucketUUIDToDBContext)
+	messageFormat := "Bucket %q is shared among databases %s. " +
+		"This may result in unexpected behaviour if security is not defined consistently."
+	for _, sharedBucket := range sharedBuckets {
+		sharedBucketError := &SharedBucketError{sharedBucket}
+		errors = append(errors, sharedBucketError)
+		base.Warnf(messageFormat, base.MD(sharedBucketError.GetSharedBucket().bucketName),
```

Comment:
```suggestion
		base.Warnf(messageFormat, base.MD(sharedBucket.bucketName),
```

Reply:
This has been committed. Changes are available in latest revision.